.github/workflows/release-apt-get.yml

```diff
@@ -55,38 +55,38 @@ jobs:
             const fs = require('fs')
             fs.writeFileSync('config.json', JSON.stringify(config, null, 2))
 
-      - name: "Get Release Asset"
```

Comment:
Is this code supposed to be deleted, or was it just temporarily commented-out for testing? If it _is_ intentionally removed, could you delete the lines rather than commenting them out?

Reply:
Ah this is what I used for testing in my fork. These lines should absolutely still be there - I amended and force pushed to origin (microsoft/git) instead of my fork (gigantic face palm). Should be all fixed now though.